packages/http/README.md

```diff
@@ -1,3 +1,376 @@
-# Prism HTTP
+# Http Client
 
-TBD
+This package provides a http client featuring the ability to:
+
+- mock responses based of OAS spec
+- make requests to a http server
+- validate the request and/or the response according to the provided OAS
+
+The goal of this document is to provide you with some basic code examples to get you started and to cover some of the advanced scenarios.
+
+# Installation
+
+`yarn add @stoplight/prism-http`
+
+# Basic Usages
+
+> Note: examples in this document use the following spec
+
+```yaml
+---
+openapi: 3.0.2
+paths:
+  /todos:
+    get:
+      parameters:
+        - name: title
+          in: query
+          style: form
+          schema:
+            enum:
+              - eat
+              - drink
+      responses:
+        200:
+          description: Get Todo Items
+```
+
+## I want to mock responses of operations defined in an OAS file
+
+[Try it out!](https://repl.it/@ChrisMiaskowski/prism-http-client-basic-mocking)
+
+```javascript
+const Prism = require('@stoplight/prism-http');
+const path = require('path');
+
+// Create Prism instance and configure it as a mocker generating static examples
+const config = { mock: { dynamic: false } };
+const prism = Prism.createInstance(config);
+
+// Load openapi spec file
+const specPath = path.resolve(process.cwd(), 'basic.oas3.yaml');
+prism
+  .load({ path: specPath })
+  .then(() => {
+    // Make a "GET /todos" request
+    return prism.process({
+      method: 'get',
+      url: {
+        path: '/todos',
+      },
+    });
+  })
+  .then(prismResponse => {
+    console.log(prismResponse.output);
+  });
+```
+
+Output
+
+```bash
+{ statusCode: 200,
+  headers: { 'Content-type': 'text/plain' },
+  body: undefined }
+```
+
+## I want to make a http request a server
+
+In this use case we assume we have a server running at `http://localhost:4010`
```

Comment:
This is a little bit confusing; this is usually the address Prism listens to. Maybe let's assume the server is running on 3030 or 8080?

Reply:
@XVincentX i was suggesting a slightly different direction, to make it even clearer, we could use not localhost.

> Instead of talking to localhost:4010 which might confuse people and make them think they are talking to Prism HTTP Server, why don't we make calls to https://pokeapi.co/ or api.example.org or literally anything other than localhost.